### PR TITLE
Add utopia to the mix.

### PR DESCRIPTION
**Pull request by ioquatix**
Add shebang lines to `bench/summary-*` scripts.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ gem "roda"
 gem "sinatra", ">= 2.0.0.rc2"
 gem "syro"
 gem "watts"
+gem "utopia"
```
Comment (luislavena) [inline Gemfile]: Please sort gems alphabetically.

File: README.md [hunk 2]
```diff
@@ -71,23 +71,24 @@ Yup, I do:
 ```
 Framework            Requests/sec  % from best
 ----------------------------------------------
-rack                     19860.11      100.00%
```
Comment (luislavena) [inline README.md]: Please, do not modify README benchmark numbers, only include new frameworks when listing the scripts but not the numbers as your hardware differs from the one used to run them in the first place.

File: apps/utopia-api.ru [hunk 3]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (luislavena) [inline apps/utopia-api.ru]: Remove unnecessary empty line

File: apps/utopia-api.ru [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+require 'utopia/controller'
+
+APP = Utopia::Controller.new(nil)
+
+class Controller < Utopia::Controller::Base
+	prepend Utopia::Controller::Actions
```
Comment (luislavena) [inline apps/utopia-api.ru]: Please use two spaces when nesting Ruby code.

File: apps/utopia-api.ru [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+	
+	on '*' do
```
Comment (luislavena) [inline apps/utopia-api.ru]: The endpoint must be `/` and respond to a `GET` request, so any other endpoint or request type must return 404.

See the Rack example for reference.

File: apps/utopia-api.ru [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+		succeed! content: 'Hello World'
+	end
+end
+
+def APP.lookup_controller(path)
+	@controller ||= Controller.new
```
Comment (luislavena) [inline apps/utopia-api.ru]: Use 2 spaces for indentation.

**Review by luislavena — CHANGES_REQUESTED** [review]
Hello,

Thank you for your PR. Please take a look to my separate comments, but to summarize:

- List any new framework alphabetically in both `README.md` and `Gemfile`
- Do not submit README's benchmark results, as your hardware differs from the one used to run them.
- Please follow Ruby's code indentation (2 spaces) and use double quotes for strings
- Ensure the test app *only* respond to `GET /` and any other request will return 404.

Cheers.

**Comment by ioquatix** [comment]
Thanks I will update.

File: Gemfile (hunk 1 above)
Comment (ioquatix) [reply Gemfile]: Done.

File: apps/utopia-api.ru (hunk 3 above)
Comment (ioquatix) [reply apps/utopia-api.ru]: Done.

File: apps/utopia-api.ru (hunk 4 above)
Comment (ioquatix) [reply apps/utopia-api.ru]: Fair enough.

File: apps/utopia-api.ru (hunk 5 above)
Comment (ioquatix) [reply apps/utopia-api.ru]: Okay done.

File: apps/utopia-api.ru (hunk 6 above)
Comment (ioquatix) [reply apps/utopia-api.ru]: Done.

File: README.md (hunk 2 above)
Comment (ioquatix) [reply README.md]: Okay I will revert.

**Pull request closed without merging**